File: public/phase-2-20160620022203/JAVASCRIPT/square.js
// Name : Square
// input:
// k :  count of cases
// an : integers a and n in every row
var idx,roots;
var k = parseInt(readline(), 10);
var an = new Array([]);
var an_elements = "";
for(idx=0;idx<k;idx++){
	an_elements = readline().split(' ');
	an[idx] = new Array([]);
	for(jdx=0;jdx<2;jdx++){
		an[idx][jdx] = parseInt(an_elements[jdx], 10);
	}
}


// write your code here
// store your results in `roots`

// Dummy Data
roots = "No root";


// output
print(roots);

File: public/phase-2-20160620022203/JAVA/square.java
// Name : Square
import java.util.*;
import java.lang.*;
import java.io.*;
class Main {
	public static void main (String[] args) throws java.lang.Exception {
		// input:
		// k :  count of cases
		// an : integers a and n in every row
		int idx;
		int jdx;
		Scanner in = new Scanner(System.in);
		int k = Integer.parseInt(in.nextLine());
		int[][] an = new int[k][2];
		String[] an_elements = new String[100];
		for(idx=0;idx<k;idx++){
			an_elements = (in.nextLine()).split(" ");
			for(jdx=0;jdx<2;jdx++){
				an[idx][jdx] = Integer.parseInt(an_elements[jdx]);
			}
		}


		// write your code here
		// store your results in `roots`

		// output
		// Dummy Data
		String roots="No root";

		System.out.println(roots);

	}
}